src/block/block_ext.c

```diff
@@ -413,6 +413,12 @@ __wt_block_off_remove_overlap(
     /* If "before" or "after" overlaps, retrieve the overlapping entry. */
     if (before != NULL && before->off + before->size > off) {
         WT_RET(__block_off_remove(session, block, el, before->off, &ext));
+        if (ext->off + ext->size < off + size) {
+            WT_BLOCK_RET(session, block, EINVAL,
+              "block off remove, befor=[%" PRIu64 ", %" PRIu64 "] overlaps with off:size=[%" PRIu64
```

Comment:
typo here "before" missing an "e"

Reply:
hi, @ruby-oujo 
thanks, I have change the code, release review again.